var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -104,6 +104,11 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         description="Enable breakpad crash dump library",
         when="@1.10:1.11",
     )
+    # py-torch has strict dependencies on old protobuf/py-protobuf versions that
+    # don't build with some of the compilers (e.g. intel) or cause problems with
+    # other packages that require newer versions of (py-)protobuf. Provide an
+    # option to use the internal/vendored protobuf instead of spack's.
+    variant("custom-protobuf", default=False, description="Use vendored protobuf/py-protobuf")
```

Comment:
```suggestion
    variant("custom-protobuf", default=False, description="Use vendored protobuf")
```
CMake describes the variant as "Build and use Caffe2's own protobuf under third_party" but I think that's even more confusing.